hack/terraform/azure/init_script_windows.ps1

```diff
@@ -0,0 +1,30 @@
+# Create temporary directory.
+$TempDir = "C:\nephe-temp\"
+New-Item -Path $TempDir -ItemType Directory
+$ScriptDir = "C:\cygwin64\tmp"
+$InstallLog = "$TempDir\terraform.log"
+
+function Log($Info) {
+    $time = $(get-date -Format g)
+    "$time $Info " | Tee-Object $InstallLog -Append | Write-Host
+}
+
+Log "Dumping with_agent ${WITH_AGENT}, namespace ${NAMESPACE}, version ${ANTREA_VERSION}"
```

Comment:
Do we want to include this script in this fixci PR?

Reply:
That was unintentional. Fixed it.